draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+| 128-255  | Specification Required
+{: #tbl-iana-comid-linked-tag-map-items-reg-procedures title="CoMID Linked Tag Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Linked-Tag Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.linked-tag-id | RFC-AAAA
+| 1 | comid.tag-rel | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-comid-linked-tag-map-items title="CoMID Linked-Tag Map Items Initial Registrations"}
+
+{: #iana-comid-triples-map-items}
+## CoMID Triples-Map Items Registry
+
+This document defines a new registry titled "CoMID Triples Map".
+The registry uses integer values as index values for items in 'comid-triples-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-triples-map-items-reg-procedures title="CoMID triples Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Triples Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.reference-triples | RFC-AAAA
+| 1 | comid.endorsed-triples | RFC-AAAA
+| 2 | comid.identity-triples | RFC-AAAA
+| 3 | comid.attest-key-triples | RFC-AAAA
+| 4-255 | Unassigned
+{: #tbl-iana-comid-triples-map-items title="CoMID Triples Map Items Initial Registrations"}
+
+{: #iana-comid-environment-map-items}
+## CoMID Environment-Map Items Registry
+
+This document defines a new registry titled "CoMID Environment Map".
+The registry uses integer values as index values for items in 'comid-environment-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-environment-map-items-reg-procedures title="CoMID Environment Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Environment Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.class | RFC-AAAA
+| 1 | comid.instance | RFC-AAAA
+| 2 | comid.group | RFC-AAAA
+| 3-255 | Unassigned
+{: #tbl-iana-comid-environment-map-items title="CoMID Environment Map Items Initial Registrations"}
+
+{: #iana-comid-class-map-items}
+## CoMID Class-Map Items Registry
+
+This document defines a new registry titled "CoMID Class Map".
```

Comment:
class-map is not extensible currently. It will break other standards if it becomes extensible. Changes should be coordinated with the specs that break otherwise.

Reply:
Just noting that `class-id` is extensible, and that is all that matters.

Reply:
flagged for pruning